docs/tutorial/deploy-first-module.md

```diff
@@ -0,0 +1,146 @@
+# Deploy your first Kit Module
+
+In this tutorial you will learn how to use collie to build and deploy landing zone infrastructure
+leveraging its integrated terraform workflow.
+
+- import a [reusable kit module](../modules/) from collie hub
+- apply kit modules to your cloud platforms and deploy
+- "bootstrapping" a cloud platform so that you can manage terraform state and 
+- safely make changes to deployed infrastructure via collie's workflow
+
+As a simple example, we will deploy an Organization Hierarchy to our cloud platform in this tutorial. Since we will
+be using Azure in this example, we will set up a Management Group structure that follows [best practices](https://cloudfoundation.org/maturity-model/tenant-management/resource-hierarchy.html).
+
+The challenges that most organizations encounter building up a landing zone are very similar. This is why the [cloudfoundation.org community](https://cloudfoundation.org)
+maintains an [open source repository](https://github.com/meshcloud/collie-hub) of reusable modules that implement solutions to these challenges here on Collie Hub. You can think of it as an equivalent of "Docker Hub" for landing zone modules.
+
+The first module that you are going to deploy on a new cloud platform is always the "boostrap" module.
```

Comment:
I feel like we could use a bit more explanation how we went from "deploy the organization hierarchy" to "oh wait, but first we need to do a bootstrap"